src/common/set.c

```
@@ -3975,6 +3975,28 @@ util_pool_open_remote(struct pool_set **setp, const char *path, int cow,
 		goto err_poolset;
 	}
 
+	/* check if there are any bad blocks */
+	int bbs = badblocks_check_poolset(set, 0 /* not create */);
+	if (bbs < 0) {
+		LOG(1,
+			"WARNING: failed to check the remote replica for bad blocks -- '%s'",
+			path);
```

Why do we ignore this error?